Diagrama Entidad Relacion

RobertoCarMed opened the pull request:
Se añadio una imagen en formato jpg en donde se plasma el diagrama entidad relacion que se debe seguir para la base de datos.

Este diagrama tiene la finalidad de servir como guia al momento de estar creando las entidades dentro de la base de datos

resolve #2 

![image](https://user-images.githubusercontent.com/79160057/137786140-008c1a44-fbe4-4a1b-8163-0afce28b3eee.png)

RobertoCarMed committed:
Diagrama Entidad Relacion

Se añadio una imagen en formato jpg en donde se plasma el diagrama entidad relacion que se debe seguir para la base de datos.

Este diagrama tiene la finalidad de servir como guia al momento de estar creando las entidades dentro de la base de datos

CUBOFIG reviewed (approved):
LGTM

xFerchoVJ reviewed (approved):
LGTM

xFerchoVJ reviewed (changes requested):
Agregar el campo Imagen para las entidades que la requieran.

RobertoCarMed committed:
Nuevo Diagrama

Se agrego el nuevo diagrama entiedad relacion en el que se agrega el campo imagen para las entidades que lo requieren

RobertoCarMed committed:
Borrar Driagrama

Se borro el antiguo diagra

xFerchoVJ reviewed (approved):
LGTM
